internal/workflow.go

```diff
@@ -1413,6 +1413,13 @@ func WithWorkflowTaskList(ctx Context, name string) Context {
 	return ctx1
 }
 
+// WithWorkflowTaskListApply returns a copy of context and changes workflow tasklist with apply function.
+func WithWorkflowTaskListApply(ctx Context, apply func(*string) *string) Context {
```

Comment:
we don't need to pass the pointer to this callback func.
`func(string) string` would be more user friendly

Reply:
make sense